packages/gravity-ui-web/src/sass/00-settings/_modularscale-settings.scss

```diff
@@ -1,7 +1,25 @@
 @import 'modularscale';
 
+
 // stylelint-disable-next-line scss/dollar-variable-pattern
 $modularscale: (
-  ratio: $major-third,
-  base: $grav-font-size-base
+  // Note: We must *only* alter the ratio across breakpoints here.
+  // We will be using this to generate CSS custom props for scaling
+  // the base font-size
+
+  ratio: $minor-second,
+  base: $grav-font-size-smallest-viewport,
+
+  $grav-breakpoint-smallest: (
+    ratio: $minor-second,
+    base: $grav-font-size-smallest-viewport,
+  ),
+
+  $grav-breakpoint-largest: (
+    ratio: $minor-third,
+    base: $grav-font-size-largest-viewport,
+  )
 );
+
+$grav-scale-min: -2;
+$grav-scale-max: 6;
```

Comment:
There's probably some good reason this is -2 to 6...?

Reply:
Yeah, I ought to add a comment.

It's based off @patrickbower's website concept designs. At the largest viewport width, the "minor thirds" scale, gives us font sizes that are pretty close the ones in the design. It just so happened that `-2` matchers the smallest size and `+6` the largest size that occured in the design.

I made them vars though, so that we can easily add _even_ bigger or smaller text sizes, should we ever need them.